### ISSUE TITLE
[Bug] shut down is working in demo

### ISSUE THREAD

**Issue by Angboo** [IceWhaleTech/CasaOS#1293]
**Describe the bug**

> use shutdown function to shutdown demo site

**To Reproduce**

> Steps to reproduce the behavior:

1. Go to [demo](http://demo.casaos.io/)
2. Click setting
3. Click shutdown
4. Wait 1min
5. The demo site has been shut down

**Expected behavior**

> Just show a alert

**Screenshots**

> If applicable, add screenshots to help explain your problem.
<img width="704" alt="image" src="https://github.com/IceWhaleTech/CasaOS/assets/4213313/036a9178-b68b-4e7e-9ca1-ec94ea47052a">

**Desktop (please complete the following information):**

```
 - OS: macos
 - Browser edge
 - Version115.0.1866.1 (正式版本) Dev(x86_64)
```

**System Time**

```
(timedatectl output here)
```

**Logs**

> Run following command to collect corresponding logs:

**Additional context**

> Add any other context about the problem here.
> 
> If you are a Zimaboard user, make it explicit with when you got your Zimaboard.

**Comment by jerrykuku** [comment]
Our demo has full CasaOS functionality. It runs on a cloud host that keeps resetting every hour. We will be monitoring its status more frequently later to maximize its uptime. Thanks for your feedback.

**Comment by ETWang1991** [comment]
@Angboo Yeah, this is a huge problem, you can turn it off every 60 minutes, CasaOS is improving the documentation, looking forward to your participation in the development of CasaOS
